server.go

```diff
@@ -606,6 +608,21 @@ func (s *Server) node() Node {
 	return *s.cluster.Node
 }
 
+// handleInstruction receives incoming Instruction from remote nodes.
+func (s *Server) handleInstruction(pb Message) {
+	instruction, ok := pb.(*Instruction)
+	if !ok {
+		s.logger.Printf("invalid message type: %T", pb)
+	}
+
+	switch instruction.Type {
+	case "ShareNodeStatus":
+		if err := s.cluster.shareNodeStatus(); err != nil {
+			s.logger.Printf("error sharing node status: %s", err)
+		}
+	}
```

Comment:
no default in this switch?

Reply:
I guess the default would be to log an error. With that said, if we remove the `Instruction` wrapper, this switch statement will go away anyway.